pact/coin-contract/v5/coin-v5.pact

```diff
@@ -437,8 +438,9 @@
              (property (valid-account miner))
              ;(property conserves-mass) not supported yet
            ]
-
-    (step (buy-gas sender total))
+    (step (let ((unused 0))
+            (validate-account miner)
+            (buy-gas sender total)))
```

Comment:
what does `unused` get us here?

Reply:
the `let` is used to group the two expressions. We have to `validate-account miner` on the first step too.

Reply:
but do we need to `let` it? Can we not write `

```
(step 
  (validate-account miner)
  (buy-gas sender total))
```
  ???

Reply:
Unfortunately not, `step` takes a single expression ...

Reply:
we really need `progn` then.

Reply:
Indeed, or better named would be `do`.

Reply:
Indeed.